lib/systems/examples.nix

```diff
@@ -98,20 +98,14 @@ rec {
   # can execute on 32bit chip
   mips-linux-gnu                = { config = "mips-linux-gnu";                } // platforms.gcc_mips32r2_o32;
```

Comment:
These also don't build ... https://hercules-ci.com/github/nix-community/cross-toolchains.nix/jobs/76

Reply:
I fixed this one.